games-roguelike/zangband/zangband-2.7.4c.ebuild

```diff
@@ -25,39 +25,41 @@ DEPEND="${RDEPEND}
 
 S=${WORKDIR}/${PN}
 
+PATCHES=( "${FILESDIR}"/${P}-tk85.patch
+	"${FILESDIR}"/${P}-rng.patch
+	"${FILESDIR}"/${P}-tinfo.patch
+	"${FILESDIR}"/${P}-configure.patch
+	"${FILESDIR}"/${P}-makefile.patch )
+
 src_prepare() {
-	epatch "${FILESDIR}"/${P}-tk85.patch \
-		"${FILESDIR}"/${P}-rng.patch \
-		"${FILESDIR}"/${P}-tinfo.patch \
-		"${FILESDIR}"/${P}-configure.patch \
-		"${FILESDIR}"/${P}-makefile.patch
+	default
+
 	mv configure.in configure.ac || die
 	eautoreconf
 }
 
 src_configure() {
-	egamesconf \
-		--datadir="${GAMES_DATADIR_BASE}" \
-		--with-setgid="${GAMES_GROUP}" \
-		--without-gtk \
+	local myconf=(
+		--libdir="${EPREFIX}"/$(get_libdir)/${PN}
+		--with-setgid="nobody"
+		--without-gtk
 		$(use_with tk tcltk)
+	)
+
+	econf "${myconf[@]}"
 }
 
-src_install() {
-	# Keep some important dirs we want to chmod later
-	keepdir "${GAMES_DATADIR}"/${PN}/lib/{apex,user,save,bone,info,xtra/help,xtra/music}
+DOCS=( readme
+	z_faq.txt
+	z_update.txt )
 
+src_install() {
 	# Install the basic files but remove unneeded crap
-	emake DESTDIR="${D}/${GAMES_DATADIR}"/${PN}/ installbase
-	rm "${D}${GAMES_DATADIR}"/${PN}/{angdos.cfg,readme,z_faq.txt,z_update.txt}
+	emake DESTDIR="${D}/" installbase
+	rm "${D}"/{angdos.cfg,readme,z_faq.txt,z_update.txt}
```

Comment:
${ED} in the latter case at least and.. || die

Reply:
To be clear: you need to always || die on external commands. rm, rmdir, anything that's not provided by PMS, and almost all eclass functions.